support/client/lib/mil-sym/cws.js

```diff
@@ -5599,104 +5848,205 @@ define( function(){
                   "tag": "BOMB AREA",
                   "symbolType": "MultiSegmentPolygon"
             },
+            // "TACGRP.FSUPP.ARS.C2ARS.PAA.RTG": {
+            //       "symbolID": "G*FPACPR--****X",
+            //       "tag": "POSITION AREA FOR ARTILLERY RECTANGULAR",
+            //       "symbolType": "RectangleMultiPoint"
+            // },
```

Comment:
Could you add a comment explaining why this is commented out?

Reply:
That is a rectangular type that I just can't get mil-sym to produce.  According to the spec, it should be defined as a line with a length modifier.  I'm faking out some other rectangle types that are supposed to be defined the same way, but they have an "irregular" type that I can define 4 cornerpoints for.  This particular one does not have that option.